components/cluster/cluster-glue/cluster-glue.p5m

```diff
@@ -0,0 +1,232 @@
+file path=usr/lib/stonith/plugins/external/ibmrsa-telnet
+file path=usr/lib/stonith/plugins/external/ipmi
+file path=usr/lib/stonith/plugins/external/ippower9258
+file path=usr/lib/stonith/plugins/external/kdumpcheck
+file path=usr/lib/stonith/plugins/external/libvirt
+file path=usr/lib/stonith/plugins/external/nut
+file path=usr/lib/stonith/plugins/external/rackpdu
+file path=usr/lib/stonith/plugins/external/riloe
+file path=usr/lib/stonith/plugins/external/ssh
+file path=usr/lib/stonith/plugins/external/vcenter
+file path=usr/lib/stonith/plugins/external/vmware
+file path=usr/lib/stonith/plugins/external/xen0
+file path=usr/lib/stonith/plugins/external/xen0-ha
+file path=usr/lib/stonith/plugins/stonith2/apcmaster.a
+file path=usr/lib/stonith/plugins/stonith2/apcmaster.so
+file path=usr/lib/stonith/plugins/stonith2/apcmastersnmp.a
+file path=usr/lib/stonith/plugins/stonith2/apcmastersnmp.so
+file path=usr/lib/stonith/plugins/stonith2/apcsmart.a
+file path=usr/lib/stonith/plugins/stonith2/apcsmart.so
+file path=usr/lib/stonith/plugins/stonith2/baytech.a
+file path=usr/lib/stonith/plugins/stonith2/baytech.so
+file path=usr/lib/stonith/plugins/stonith2/cyclades.a
+file path=usr/lib/stonith/plugins/stonith2/cyclades.so
+file path=usr/lib/stonith/plugins/stonith2/drac3.a
+file path=usr/lib/stonith/plugins/stonith2/drac3.so
+file path=usr/lib/stonith/plugins/stonith2/external.a
+file path=usr/lib/stonith/plugins/stonith2/external.so
+file path=usr/lib/stonith/plugins/stonith2/ibmhmc.a
+file path=usr/lib/stonith/plugins/stonith2/ibmhmc.so
+file path=usr/lib/stonith/plugins/stonith2/meatware.a
+file path=usr/lib/stonith/plugins/stonith2/meatware.so
+file path=usr/lib/stonith/plugins/stonith2/null.a
+file path=usr/lib/stonith/plugins/stonith2/null.so
+file path=usr/lib/stonith/plugins/stonith2/nw_rpc100s.a
+file path=usr/lib/stonith/plugins/stonith2/nw_rpc100s.so
+file path=usr/lib/stonith/plugins/stonith2/rcd_serial.a
+file path=usr/lib/stonith/plugins/stonith2/rcd_serial.so
+file path=usr/lib/stonith/plugins/stonith2/rhcs.a
+file path=usr/lib/stonith/plugins/stonith2/rhcs.so
+file path=usr/lib/stonith/plugins/stonith2/ribcl.py
+file path=usr/lib/stonith/plugins/stonith2/rps10.a
+file path=usr/lib/stonith/plugins/stonith2/rps10.so
+file path=usr/lib/stonith/plugins/stonith2/ssh.a
+file path=usr/lib/stonith/plugins/stonith2/ssh.so
+file path=usr/lib/stonith/plugins/stonith2/suicide.a
+file path=usr/lib/stonith/plugins/stonith2/suicide.so
+file path=usr/lib/stonith/plugins/stonith2/wti_mpc.a
+file path=usr/lib/stonith/plugins/stonith2/wti_mpc.so
+file path=usr/lib/stonith/plugins/stonith2/wti_nps.a
+file path=usr/lib/stonith/plugins/stonith2/wti_nps.so
+file path=usr/lib/stonith/plugins/xen0-ha-dom0-stonith-helper
+file path=usr/sbin/cibsecret
+file path=usr/sbin/ha_logger
+file path=usr/sbin/hb_report
+file path=usr/sbin/lrmadmin
+file path=usr/sbin/meatclient
+file path=usr/sbin/stonith
+file path=usr/share/cluster-glue/ha_cf_support.sh
```

Comment:
I determine now with a little helper script all files set executable by make install. 
This script outputs the lines in p5m format with mode=0555.
This is better than just use transformations for folders.